src/components/Footer/Footer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <>
+     <div className='footer'>
+        <a class="social-media-link" href="https://github.com/shrushti2000"> <i class="fa fa-github"></i></a>
```

Comment:
It would be a great UX if you could add `target="_blank"`. This would ensure your external links are opened in a new tab and user will not have to use back button to navigate back to your website!